Formula/ruby.rb

```diff
@@ -31,8 +31,8 @@ class Ruby < Formula
   # The exception is Rubygem security fixes, which mandate updating this
   # formula & the versioned equivalents and bumping the revisions.
   resource "rubygems" do
-    url "https://rubygems.org/rubygems/rubygems-2.7.6.tgz"
-    sha256 "67f714a582a9ce471bbbcb417374ea9cf9c061271c865dbb0d093f3bc3371eeb"
+    url "https://rubygems.org/rubygems/rubygems-2.7.7.tgz"
```

Comment:
Should this be updating rubygems at the same time?

Reply:
> __Should be updated only when Ruby is updated (if an update is available).__
> The exception is Rubygem security fixes, which mandate updating this
> formula & the versioned equivalents and bumping the revisions.